File: bench/fixtures/feeds/python/stages.py
"""The pipeline stages, in two independent pairs so the plain and hidden flow
tests do not share a candidate pool."""


def parse(text: str) -> int:
    return len(text)


def render(n: int) -> str:
    return str(n)


def collect(text: str) -> int:
    return len(text)


def format_(n: int) -> str:
    return str(n)

File: bench/fixtures/feeds/typescript/stages.ts
// The pipeline stages, in two independent pairs so the plain and hidden flow
// tests do not share a candidate pool.
export function parse(text: string): number {
  return text.length;
}

export function render(n: number): string {
  return String(n);
}

export function collect(text: string): number {
  return text.length;
}

export function format(n: number): string {
  return String(n);
}
